### PR TITLE
Editing of "Constructor" part of the tutorial

### PR DESCRIPTION
**Pull request by BJackal**
I think there was an error saying to add to the "construct" function as new bdm .h files do not come with any function named "construct", I think saying to add the changes to the constructor instead is a little more clear.

### CODE REVIEW

**Comment by breitwieserCern** [comment]
Hey Jack, 

You are right, the `construct` function does not exist. Seems like documentation got out of synch with our codebase. (It only exists in the `cell_division_module` example [1]). 

What do you think if we change it to: "And add the following line to the `Simulate` function after the cell was created"? 
Because the line is not added to a class constructor.

[1] https://github.com/BioDynaMo/biodynamo/blob/9473c9eacb6f634a93f610d1b86c3003a6aed98d/demo/cell_division/src/cell_division_module.h#L41-L48

**Comment by BJackal** [comment]
Hi Lukas, 

I believe that would be an excellent way of re-wording the tutorial as it makes it much more clear overall for the user.

**Comment by breitwieserCern** [comment]
Great! Could you please update the PR?

File: doc/user_guide/docs/hello_world.md [hunk 1]
```diff
@@ -50,11 +50,17 @@ BDM_CTPARAM() {
...
 ``` C++
-cell.AddBiologyModule(GrowDivide(32, 3000, {gAllEventIds}));
+auto construct = [](const std::array<double, 3>& position) { 
+   Cell cell(position);  
+   cell.SetDiameter(30); 
+   cell.SetAdherence(0.4); 
+   cell.SetMass(1.0); 
+   cell.AddBiologyModule(GrowDivide()); # Adding the GrowDivide biology modules behaviour to the cell.
+   return cell; 
+};
```
Comment (breitwieserCern) [inline doc/user_guide/docs/hello_world.md]: Sorry it seems I wasn’t clear about the code example from my comment before [1]. I added it only to explain where the phrase "construct function" originated from.
Could you please change it back to `cell.AddBiologyModule(GrowDivide(32, 3000, {gAllEventIds}));` 

[1] https://github.com/BioDynaMo/biodynamo/pull/45#issuecomment-451939853

**Comment by breitwieserCern** [comment]
Thanks Jack!

**Pull request merged**